core/pipeline/label_test.go

```diff
@@ -2,13 +2,134 @@ package pipeline_test
 
 import (
 	"os"
+	"os/exec"
 	"strings"
 	"testing"
 
 	"github.com/dagger/dagger/core/pipeline"
 	"github.com/stretchr/testify/require"
 )
 
+func run(t *testing.T, exe string, args ...string) string {
```

Comment:
I assume that `exe string` is not necessary here and `args ...string` should be enough ?

Reply:
`exec.Command` has a required argument for the command, followed by variadic args, so this function just reflects that. Otherwise it'd be possible to construct an "empty" command. :)

Reply:
gotcha